compose-oneliner.sh

```diff
@@ -233,6 +244,7 @@ case "${PRODUCT}" in
     ;;
     "insights")
         timedatectl set-timezone Etc/UTC && echo "changed the local machine time to UTC"
+        run_xhost
```

Comment:
make it 'xhost +SI:localuser:root' insted of 'xhost +'

Reply:
fixed in the function